src/config.rs

```diff
@@ -68,20 +72,25 @@ pub struct DigitalOutput {
     pub gpio: u32,
     pub mqtt_topic: String,
     pub mqtt_topic_set: String,
-    #[serde(default = "DigitalOutput::default_mqtt_state_on")]
-    pub mqtt_state_on: String,
-    #[serde(default = "DigitalOutput::default_mqtt_state_off")]
-    pub mqtt_state_off: String,
+    #[serde(default = "DigitalOutput::default_mqtt_state_high")]
+    pub mqtt_state_high: String,
+    #[serde(default = "DigitalOutput::default_mqtt_state_low")]
+    pub mqtt_state_low: String,
+    #[serde(default = "DigitalOutput::default_initial_state")]
```

Comment:
If I didn't miss it: I think `initial_state` is not documented in config.yaml yet, right?

Reply:
Done.